PathPlanning/TimeBasedPathPlanning/GridWithDynamicObstacles.py

```diff
@@ -0,0 +1,278 @@
+import numpy as np
```

Comment:
Please add header comment to explain this script.

Reply:
Added a short header describing the class